spec/features/admin_deletes_user_spec.rb

```diff
@@ -1,33 +1,26 @@
 require 'rails_helper'
 
-feature 'admin deletes user', %Q{
+feature 'admin deletes article', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.